exporter/postgres/crunchy-postgres-exporter-pg10-el6.service

```diff
@@ -35,7 +35,7 @@ start() {
 #  do_start_prepare
 
   echo -n $"Starting postgres_exporter: "
-  cat $QUERY_FILE_LIST | sed "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES}/g" > $(echo $OPT | sed 's/.*--extend.query-path=\(.*\) .*/\1/')
+  cat $QUERY_FILE_LIST | sed "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES}/g" > $(echo $OPT | sed 's/.*--extend.query-path=\(.*\.yml\).*/\1/')
```

Comment:
This is already merged via https://github.com/CrunchyData/pgmonitor/pull/114 ?

Reply:
Yes, so this part will just do nothing then, right?